eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -388,9 +392,6 @@ jobs:
         CopyWithRelativeFolders "artifacts/"                 $targetFolder "*.diff"
         CopyWithRelativeFolders "src/"                       $targetFolder "*.binlog"
         CopyWithRelativeFolders "src/"                       $targetFolder "*.log"
-        CopyWithRelativeFolders "test/"                      $targetFolder "*.binlog"
-        CopyWithRelativeFolders "test/"                      $targetFolder "Updated*.diff"
-        CopyWithRelativeFolders "test/"                      $targetFolder "Updated*.txt"
```

Comment:
How is this handled with these changes?

Reply:
Tests previously just put logs into the cwd. I just pushed a commit to respect a given log directory and updated the YML to still upload the "Updated*.txt" files.